src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
 body {
+  background-color: #f0f0f0;
   margin: 0;
+  padding: 50px;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  color: #060b35;
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__product-code {
+  color: #616070;
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__reviews {
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  display: flex;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
```

Comment:
The `background-image` URL should be relative to the `index.scss` file. According to the task requirements, it should start with `../images` instead of `/src/images`【8:2†description.md】.